MCprep_addon/materials/generate.py

```diff
@@ -666,6 +663,12 @@ def replace_missing_texture(image: Image) -> bool:
 	image.filepath = str(image_path)
 	# image.reload() # not needed?
 	# pack?
+	# Due to the issue below, must trick blender to reload the datablock
+	# https://projects.blender.org/blender/blender/issues/115984
+	if image.source == 'FILE' and image.source != 'SEQUENCE':
```

Comment:
The check for SEQUENCE was necessary as it was otherwise causing a ripple effect downstream for some reason.